starknet_py/contract.py

```diff
@@ -370,9 +376,17 @@ def functions(self) -> FunctionsRepository:
     def address(self) -> int:
         return self._data.address
 
+    @property
+    def data(self) -> ContractData:
+        return self._data
+
     @staticmethod
     async def from_address(
-        address: AddressRepresentation, client: Client
+        address: AddressRepresentation,
+        client: Client,
+        resolve_proxies: bool = False,
```

Comment:
This adds too much overhead - i'd rather have a typed dict named `proxy_config` with some reasonable defaults